tools/lint

```diff
@@ -47,9 +47,12 @@ def run():
     parser.add_option('--full',
                       action='store_true',
                       help='Check some things we typically ignore')
-    parser.add_option('--pep8',
-                      action='store_true',
-                      help='Run the pep8 checker')
+    parser.add_option('--py',
+                      action="store_true",
+                      help='Don\'t run expensive Javascript checks')
+    parser.add_option('--js',
+                      action="store_true",
+                      help='Don\'t run expensive Python checks')
```

Comment:
I think structurally, I'd rather have this work off of excluding certain extensions, not excluding certain linters.  The effect will be the same for these linters, but it'd also save running things like the CSS linter as well.